src/engraving/libmscore/dynamic.h

```diff
@@ -86,10 +54,8 @@ class Dynamic final : public TextBase
         const char* name;
     };
 
-    Q_ENUM(Type);
```

Comment:
Isn't thsat needed to make them available to plugins?

Reply:
I'll take care of this in a separate PR